config_db.yml

```diff
@@ -0,0 +1,35 @@
+databricks_pipelines:
```

Comment:
Can we change this file to config_databricks? Because db makes me think of database instead